cfy_manager/components/logstash/logstash.py

```diff
@@ -184,8 +88,7 @@ def configure():
 
 def remove():
     logger.notice('Removing Logstash...')
-    remove_notice(LOGSTASH)
-    systemd.remove(LOGSTASH)
-    remove_files([HOME_DIR, LOG_DIR, UNIT_OVERRIDE_PATH, LOGSTASH_CONF_DIR])
-    yum_remove(LOGSTASH)
+    systemd.remove(LOGSTASH, service_file=False)
+    for package in LOGSTASH, 'postgresql-jdbc', 'cloudify-logstash':
```

Comment:
do we need to remove all of them? can we just remove the one the others depend on?